feature-libs/asm/components/customer-list/customer-list.component.html

```diff
@@ -13,7 +13,12 @@ <h2 id="asm-customer-list-title" class="title modal-title">
         <div id="asm-customer-list-desc" class="cx-visually-hidden">
           {{ 'asm.customerList.description' | cxTranslate }}
         </div>
-        <ng-template *ngTemplateOutlet="closeButton"></ng-template>
```

Comment:
Please add a TODO command to remove the feature flag associated with the Jira ticket number.

Reply:
done